service/collector/elb.go

```diff
@@ -120,20 +120,27 @@ func (e *ELB) collectForAccount(ch chan<- prometheus.Metric, awsClients clientaw
 		return microerror.Mask(err)
 	}
 
-	var loadbalancers []*elb.LoadBalancerDescription
+	var loadBalancers []*elb.LoadBalancerDescription
```

Comment:
While here, I spotted a missing capitalization for variable naming scheme here...